back/app/controllers/web_api/v1/projects_controller.rb

```diff
@@ -76,6 +76,46 @@ def create
     end
   end
 
+  def copy
+    source_project = Project.find(params[:id])
+
+    new_title_multiloc = add_suffix_to_title(source_project.title_multiloc)
```

Comment:
I believe we could move all this code to a separate service with a name like `LocalProjectCopyService`.

The action won't be fat, and the service will be easy to test. You'll write just one simple acceptance test, and the rest will be covered by the service unit tests.

Reply:
OK. Sounds good.